docs/sources/developers/plugins/build-a-streaming-data-source.md

```diff
@@ -0,0 +1,132 @@
++++
+title = "Build a streaming data source"
+type = "docs"
++++
+
+# Build a streaming data source
+
+> This guide assumes that you're already familiar with how to [Build a data source plugin]({{< relref "../../../../../tutorials/build-a-data-source-plugin.md" >}}).
+
+When monitoring critical applications, you want your dashboard to refresh as soon as your data does. In Grafana, you can set your dashboards to automatically refresh at a certain interval, no matter what data source you use. Unfortunately, this means that your queries are requesting all the data to be sent again, regardless of whether the data has actually changed.
```

Comment:
This is not a great first paragraph, so I suggest making a new one. Something like,

This guide explains how to set up your plugin to work with a streaming data source.